Frontend/ui-driver/src/Services/Backend.purs

```diff
@@ -184,7 +184,7 @@ makeTriggerOTPReq    mobileNumber = TriggerOTPReq
     {
       "mobileNumber"      : mobileNumber,
       "mobileCountryCode" : "+91",
-      "merchantId" : if SC.getMerchantId == "NA" then getValueToLocalNativeStore MERCHANT_ID else SC.getMerchantId
+      "merchantId" : if (SC.getMerchantId "") == "NA" then getValueToLocalNativeStore MERCHANT_ID else (SC.getMerchantId "" )
```

Comment:
These brackets were not required here.

@kavya-shree-s please make sure redundant code is not merged. If doing so add comments please